src/s2wasm.h

```diff
@@ -153,9 +153,16 @@ class S2WasmBuilder {
 
   Name getStr() {
     std::string str; // TODO: optimize this and the other get* methods
-    while (*s && !isspace(*s)) {
-      str += *s;
-      s++;
+    if (*s == '\"') {
```

Comment:
This if-block is repeated 3 times here and below; can we factor it into a function? Any other common code from those uses that can also be factored out?